src/resources/ECForms.ts

```diff
@@ -108,14 +108,15 @@ export type ResponseECForm<Metadata = BaseMetadata> = ECFormItem<Metadata>;
 
 export class ECForms extends CRUDResource {
     static requiredParams: string[] = [];
-    static routeBase = "/forms";
+    static routeBase = "/merchants/:merchantId/forms";
```

Comment:
We do not have authorization headers in the form checkout at the point of querying this. Moving it in the parameters instead.